lca-cli/postpivot/postpivot.go

```diff
@@ -160,11 +163,20 @@ func (p *PostPivot) PostPivotConfiguration(ctx context.Context) error {
 	}
 
 	// Restore OADP secrets and DPA if exists
-	if err := p.restoreOadpSecrets(ctx, client); err != nil {
-		return fmt.Errorf("failed to restore OADP secrets: %w", err)
-	}
-	if err := p.restoreOadpDataProtectionApplication(ctx, client); err != nil {
-		return fmt.Errorf("failed to restore OADP DataProtectionApplication: %w", err)
+	ibu := &lcav1alpha1.ImageBasedUpgrade{}
+	filePath := common.PathOutsideChroot(utils2.IBUFilePath)
+	if err := utils.ReadYamlOrJSONFile(filePath, ibu); err != nil {
+		return fmt.Errorf("failed to read locally saved IBU CR: %w", err)
+	}
+	if len(ibu.Spec.OADPContent) > 0 {
+		if err := p.restoreOadpSecrets(ctx, client); err != nil {
+			return fmt.Errorf("failed to restore OADP secrets: %w", err)
+		}
+		if err := p.restoreOadpDataProtectionApplication(ctx, client); err != nil {
+			return fmt.Errorf("failed to restore OADP DataProtectionApplication: %w", err)
+		}
+	} else {
+		p.log.Info("Locally stored IBU CR did not have any OADPContent")
```

Comment:
The check is not needed. If the directores don't exist or don't have files, it will just skip.

Reply:
reverted